starter-code/pizza.js

```diff
@@ -1 +1,66 @@
 // Write your Pizza Builder JavaScript in this file.
+
+'use strict'
+
+
+function main () {
+
+var $div = $('<div>Hiii</div>');
+var $body = $('body');
+$body.append($div);
+
+
+var $pizza = $('#pizza section');
+var $pepperonnis = $('.pep');
+var $greenPeppers = $('.green-pepper');
+var $mushrooms = $('.mushroom');
+var $crustGlutenFree = $('.crust-gluten-free');
+var $sauceWhite = $('.sauce-white');
+
+
+var $button = $('.btn');
+var $pepperonnisButton = $('.btn-pepperonni');
+var $greenPeppersButton = $('.btn-green-peppers');
+var $mushroomsButton = $('.btn-mushrooms');
+
+
+
+$crustGlutenFree.removeClass('.crust-gluten-free');
+$sauceWhite.removeClass('.sauce-white');
```

Comment:
These doesn't work because you don't have to put the dot (.) when the method especifies that works with a class.